sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -763,6 +763,10 @@ def _download_blob_options(self, offset=None, length=None, encoding=None, **kwar
         validate_content = kwargs.pop('validate_content', False)
         access_conditions = get_access_conditions(kwargs.pop('lease', None))
         mod_conditions = get_modify_conditions(kwargs)
+        if 'version_id' in kwargs:
+            version_id = kwargs.pop('version_id')
+        else:
+            version_id = self.version_id
```

Comment:
I wonder if it's worth putting these four lines in a helper function somewhere? I would need to take in `kwargs` and anoterh version_id param where we pass `self.version_id` so wouldn't be to much better but then it could be on one line in all the places its used (which is a lot).